README.md

```diff
@@ -26,127 +26,80 @@ MDC Web strives to seamlessly incorporate into a wider range of usage contexts,
 
 ## Quick start
 
-Install the library
+> Note: This guide assumes you have npm installed locally.
 
-```
-npm install --save material-components-web
-```
-
-Then simply include the correct files, write some HTML, and call `mdc.autoInit()` within a closing
-`<script>` tag.
-
-```html
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Material Components for the web</title>
-    <link rel="stylesheet"
-          href="node_modules/material-components-web/dist/material-components-web.css">
-  </head>
-  <body class="mdc-typography">
-    <h2 class="mdc-typography--display2">Hello, Material Components!</h2>
-    <div class="mdc-text-field" data-mdc-auto-init="MDCTextField">
-      <input type="text" class="mdc-text-field__input" id="demo-input">
-      <label for="demo-input" class="mdc-text-field__label">Tell us how you feel!</label>
-    </div>
-    <script src="node_modules/material-components-web/dist/material-components-web.js"></script>
-    <script>mdc.autoInit()</script>
-  </body>
-</html>
-```
-
-That's all there is to it! This is the easiest way to get up and running with Material Components
-for web. Check out our [Getting Started guide](./docs/getting-started.md) for a more in-depth
-introduction to the library.
+### Include CSS for a component
 
-## Installing individual components
+> Note: This guide assumes you have webpack configured to compile Sass into CSS. See this [getting started guide](docs/getting-started.md) for pointers on how to configure webpack.
 
-MDC Web is modular by design. Each component lives within its own packages under the
-[@material npm org](https://www.npmjs.com/org/material).
+To include the Sass files for the Material Design button, install the Node dependency:
 
 ```
-npm install --save @material/button @material/card @material/textfield @material/typography
+npm install @material/button
 ```
 
-All our components can be found in the [packages](./packages) directory. Each component has a
-README documenting installation and usage.
+Then import the Sass files for @material/button into your application. You can also use Sass mixins to customize the button:
 
-## Including components
-
-### JavaScript
-
-If you are using a module loader such as Webpack or SystemJS to load your JS modules, you can simply
-`import` every component you need from `material-components-web` and use it as such.
-
-```js
-import {checkbox as mdcCheckbox} from 'material-components-web';
-
-const {MDCCheckbox, MDCCheckboxFoundation} = mdcCheckbox;
-// Use MDCCheckbox and/or MDCCheckboxFoundation
 ```
+@import "@material/button/mdc-button";
```

Comment:
Specify scss on this fenced code block (right after the opening triple backtick) for syntax highlighting